src/Pos/WorkMode.hs

```diff
@@ -17,6 +17,8 @@ module Pos.WorkMode
 import           Universum
 
 import           Control.Monad.Fix
+import           Control.Monad.Base             (MonadBase)
+import           Control.Monad.Trans.Control
```

Comment:
Currently we do it by hand, but we have a group of people (actually only two people) working on a separate utility to facilitate work with imports.

> If it's the former, that sounds quite inefficient.

It makes writing code less efficient, but reading code more efficient. The latter is usually more important.